odoo_sentinel/__init__.py

```diff
@@ -81,9 +84,6 @@ def __init__(self, stdscr, options):
             language = gettext.translation(
                 I18N_DOMAIN, I18N_DIR, languages=[I18N_DEFAULT])
 
-        # Replace global dummy lambda by the translations gettext method
-        # The install method of gettext doesn't replace the function if exists
-        global _
         _ = language.gettext
```

Comment:
You should move this line too, because the translation method is called on line 67.